Model/Sales/MagentoOrderCollection.php

```diff
@@ -329,11 +329,18 @@ public function updateMagentoTrack(): self
                     $magentoTrack->getData('myparcel_consignment_id')
                 );
 
-                $magentoTrack->setData('myparcel_status', $myParcelTrack->getStatus());
+                if (! $myParcelTrack) {
+                    continue;
+                }
 
-                if ($myParcelTrack->getBarcode()) {
+                if (! empty($myParcelTrack->getStatus())) {
```

Comment:
Empty en ! zijn elkaars exact tegenovergestelde, is dan ```if ($myParcelTrack->getStatus()) {``` niet precies hetzelfde?

Reply:
Volgens mij niet want bij `if ($myParcelTrack->getStatus()) {` kan de waarde ook `""` zijn en als je `if (! empty($myParcelTrack->getStatus())) {` gebruikt dan moet er echt iets in zitten

Reply:
Het lijkt inderdaad te werken, ik ga het aanpassen

Reply:
Lege string is falsy idd, ik ben wel voor alleen `empty()` gebruiken als het écht nodig is